tests/regression/rhbz-1584461/task.yaml

```diff
@@ -20,8 +20,7 @@ restore: |
 
 execute: |
     # on Fedora, /etc/nsswitch.conf is a symlink to /etc/authselect/nsswitch.conf
-    # TODO: uncomment once we fix this in spread images
-    # test -s /etc/nsswitch.conf
+    test -h /etc/nsswitch.conf
```

Comment:
Silly question - why `test -h` instead of `test -L` ? Mostly asking because 'L' -> 'link' in my brain but 'h' -> 'manpage'. Anyway, not a real issue, mostly curious.

Reply:
Hahah, I can update it to -L